doc/man1/pkeyutl.pod

```diff
@@ -144,12 +143,12 @@ hex dump the output data.
 
 =item B<-asn1parse>
 
-asn1parse the output data, this is useful when combined with the
+Asn1parse the output data, this is useful when combined with the
```

Comment:
This kind of doesn't make sense. "asn1parse" is an option name and shouldn't have a capital letter. Here though it seems to be being used as a verb.